schema/ordering.proto

```diff
@@ -11,4 +12,5 @@ service OrderingGateTransportGrpc {
 
 service OrderingServiceTransportGrpc {
   rpc onTransaction (iroha.protocol.Transaction) returns (google.protobuf.Empty);
+  rpc onBatch (iroha.protocol.TxList) returns (google.protobuf.Empty);
```

Comment:
Why it is here btw? I thought onBatch will be implemented in a separate PR

Reply:
Because otherwise gRPC call method will not be generated. It's needed in ordering_gate_transport_grpc.cpp, line 89